### ISSUE TITLE
Show paths/controls for both panes in split view

### ISSUE THREAD

**Issue by AndreiMiculita** [linuxmint/nemo#3046]
Currently, opening a split view with F3 only splits the main view (with the icons/thumbnails). It would be very useful if the path toolbar was also split somehow to display paths for each pane. Ideally, the controls (Back/Forward/Up/Refresh) would also be duplicated but that might be a bit crowded.

Been working a lot with the split view recently and I'm definitely missing this, it's easy to forget the path of the unfocused view, so then it requires 2 more clicks to check it and then focus back on the initial view.

![image](https://user-images.githubusercontent.com/6352172/183440140-60692c2c-b43b-47e6-8370-80dbd1d89e92.png)

**Comment by Jeremy7701** [comment]
Also,  it would be very useful to have a stronger indication of which pane has the focus - perhaps different backgrounds?

**Comment by AndreiMiculita** [comment]
> Also, it would be very useful to have a stronger indication of which pane has the focus - perhaps different backgrounds?

In the default theme at least, that's already done. Notice how in my screenshot the right side is brighter.

**Comment by AndreiMiculita** [comment]
For an example, check out the way Total Commander from Windows does it. Both panes have their own path visible:

![image](https://user-images.githubusercontent.com/6352172/183482010-2a8b5e10-953e-4745-a3ca-0010143a4272.png)

**Comment by Drugwash2** [comment]
@AndreiMiculita You might be better off with **Double Commander** instead of Nemo if you want aspect and behavior as close to Total Commander as possible. Although DC has a few quirks and shortcomings it is highly configurable, more so than TC and infinitely more than Nemo.

I myself tried to get accustomed to Nemo coming from 20+ years of TotCmd but just couldn't, so now I went over to Double Commander and slowly getting to feel like back home.

As for the focused pane distinction @Jeremy7701 that is indeed up to the current theme, as pointed out by Andrei. See here Nemo in Cinnamon 4.2 (Mint 19.2) where focused pane (left) is brighter than the other one:
![Screenshot from 2022-08-09 02-32-42](https://user-images.githubusercontent.com/29117024/183533549-74a103c9-c6f5-43db-a727-8f807ab4041c.png)

**Comment by Jeremy7701** [comment]
Using a  Mint-X light theme on LMDE5 (Cinnamon 5.27) fails to provide this kind of highlighting.
I could use a Mint-Y Control theme but then the permissions flags are no longer shown as tri-state flags.

**Comment by Jeremy7701** [comment]
Just got the update for Cinnamon LMDE5 and all problems are fixed.

**Comment by AndreiMiculita** [comment]
> You might be better off with Double Commander instead of Nemo if you want aspect and behavior as close to Total Commander as possible.

Agree, but I've never used Total Commander as a daily driver and I'm not looking for a recreation of it. It's just that splitting panes is a feature that I find a bit incoherent in Nemo as it is at the moment.

> Using a Mint-X light theme on LMDE5 (Cinnamon 5.27) fails to provide this kind of highlighting.

Indeed, also seeing this. That's issue with Mint X itself.

**Comment by Drugwash2** [comment]
> Agree, but I've never used Total Commander as a daily driver [...]

Since day one I realized TC's superiority to Explorer and never looked back. Granted plug-ins give it a lot of extra power but same goes with Double Commander. It may be an acquired taste but once gotten it will never leave you. It's not for everybody though.
Nemo can't hold a candle to DC.

**Comment by AndreiMiculita** [comment]
> I wouldn't mind adding something that might at least show the current location for each pane, but it's not something we've spent any time on.

A read-only label containing the path could be useful. Or, to make it more minimalistic, only the name of the directory itself. With some extra info if we need to differentiate.

For example, here's what an Intellij IDE does:

* When a file is open, show its name in the tab title:
![image](https://user-images.githubusercontent.com/6352172/191459799-e1b9d78a-3c80-416f-b06d-c8a3882b4fcb.png)
* When two files with the same name, but different locations are open, show as much of their path as needed to differentiate in the tab title. Here for example they only need to go 1 dir up:
![image](https://user-images.githubusercontent.com/6352172/191460007-8ed8c265-f9f9-47d4-93ad-2442ccc77880.png)

So if we have these directories:

1. foo/bar/baz
2. foo/bar/qux
3. foo/quux/baz
4. quuz/bar/baz

|Open directories | Pane labels |
|---|---|
|1 and 2|`baz` and `qux`|
|1 and 3|`bar/baz` and `quux/baz`|
|1 and 4|`foo/.../baz`and `quuz/.../baz`|

Also, using an ellipsis for the identical parts would trim it down even more.

**Comment by Eichhorst** [comment]
Currently i'm using Caja (on MATE) only because of this (missing) feature!
I'd be happy to see it implemented in Nemo, too!

**Comment by cdmdotnet** [comment]
I'd just be thrilled if you could toggle between horizontal vs vertical panes and a toggle to have a single/combined left pan (tree/places) or dual left panes like xplorer2 has. 

I think this would polish the dual pane feature of nemo off perfectly.

<img width="1204" height="1279" alt="Image" src="https://github.com/user-attachments/assets/bcfdc4e4-c56e-4b60-ba03-65654fd2b54e" />

**Comment by mtwebster** [comment]
Addressing the original issue, the controls are too crowded if you attempt to have two sets - this was something Nemo used to have, but was removed a long time ago. There are no plans for its return. OP's screenshot looks like a 4k monitor in full resolution - this is not a typical situation for most users.

**Comment by AndreiMiculita** [comment]
> OP's screenshot looks like a 4k monitor in full resolution - this is not a typical situation for most users.

If only there was some way to tell what resolution my screen was in that picture....

Hint: it very much was not 4K. It's 4 times smaller. And the total commander screenshot is even smaller.

<img width="2126" height="998" alt="Image" src="https://github.com/user-attachments/assets/e190b512-ebe1-455c-ad39-37b772cef82b" />

**Comment by Peeder78** [comment]
I would very much like to have this feature, turned off by default and settable via options.
 
> Addressing the original issue, the controls are too crowded if you attempt to have two sets
Nowadays, I open two instances of nemo and put them side by side, using automatic window tiling. There, two sets of controls fit without a problem side by side (however, I would only like to have the path, not the rest of the buttons).
